src/js/index.js

```diff
@@ -452,7 +452,33 @@ function init(config) {
 function getState() {
   return new Promise(async (resolve, reject) => {
     await checkUserEnvironment().catch(reject)
-    resolve(state)
+    const {
+      mobileDevice,
+      validBrowser,
+      currentProvider,
+      web3Wallet,
+      accessToAccounts,
+      walletLoggedIn,
+      walletEnabled,
+      accountAddress,
+      accountBalance,
+      userCurrentNetworkId,
+      correctNetwork
+    } = state
```

Comment:
Should we also be returning `minimumBalance` here?

Reply:
Good spot! Has been added in 845a257